lms/djangoapps/learner_dashboard/tests/test_programs.py

```diff
@@ -0,0 +1,138 @@
+"""
+Tests for viewing the programs enrolled by a learner.
+"""
+import datetime
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.test import override_settings
+import httpretty
+from opaque_keys.edx import locator
+import unittest
+from urlparse import urljoin
```

Comment:
An upgraded version of Pylint would have caught this: please organize your imports into the three standard groups (standard library, third-party, first-party).